apps/i18n/studio/en_us.json

```diff
@@ -378,6 +378,7 @@
   "repeatDo": "do",
   "repeatForeverTooltip": "Execute the actions in this block repeatedly while the story is running.",
   "spaceInvasion": "Space invasion!",
+  "foodFight": "Food fight!",
```

Comment:
should these (titles?) be grouped with similar text?  they seem a bit unusual right here..